src/GeolocationContext.tsx

```diff
@@ -135,6 +135,10 @@ export const GeolocationContextProvider: React.FC = ({children}) => {
   const geoLocationName = t(dictionary.myPosition); // TODO: Other place for this fallback
   const currentCoordinatesRef = useRef<Coordinates | undefined>();
   const {updateMetadata} = useIntercomMetadata();
+  Geolocation.setRNConfiguration({
+    locationProvider: 'playServices',
+    skipPermissionRequests: true,
+  });
```

Comment:
Should this be set outside of the `GeolocationContextProvider`? Now this is getting called every render.